### ISSUE TITLE
Miniflux:  Menu Shows Multiple of the Same Category rather than just one

### ISSUE THREAD

**Issue by CtrlAltDelete983422** [yang991178/fluent-reader#70]
**Describe the bug**
Menu shows multiple categories rather than showing just one category

**To Reproduce**
1. Launch Fluent Reader.exe
2. Configure Service via Fever API for first sync
3. After first sync complete, click on Menu button in upper left hand corner
4. Some of the categories are all rolled up into one category but others show multiple categories listed and only contain a single RSS feed per category
5. See screen shots for better clarification

**Expected behavior**
Expect to only have one category with multiple RSS feeds within each category in the Menu

**Screenshots**
![Fluent_Reader_pwdbubhHIJ](https://user-images.githubusercontent.com/69652380/90188124-71b3a980-ddaa-11ea-8b54-ceaed262652c.png)
![Fluent_Reader_6RmAnA0X0i](https://user-images.githubusercontent.com/69652380/90188391-e981d400-ddaa-11ea-98a2-e5dfb8dee473.png)

**Platform (please complete the following information):**
 - OS: Windows 10 2004 (OS Build 19041.450)
 - Version 0.7.2

**Additional context**
Using Miniflux via Fever API to connect; Sync Limit set to 1000 latest articles

**Comment by yang991178** [comment]
Please check if the group names contain trailing spaces. Thank you for reporting this.

**Comment by CtrlAltDelete983422** [comment]
You were correct.  I had spaces (one before on one category and one after on the other two categories in question).  I updated the categories within Miniflux.  I did a sync but it did not update the categories within Fluent Reader (There may be a way to do it, I just wasn't sure how).  I started over and everything is now showing up correctly.  My apologies for the false positive.  Thank you for looking into it for me.
